test_runner.py

```diff
@@ -57,10 +57,12 @@ def build_test_list():
             [
                 [
                     "--training.compile",
+                    "--training.enable_cpu_offload True",
```

Comment:
Add into 1D compile test first

Reply:
sorry I didn't make it clear during offline discussion.
I think we shouldn't touch this test. Instead we should create a new one with `enable_cpu_offload` only